css/amp.css

```diff
@@ -93,6 +93,13 @@ html.i-amphtml-ios-embed {
   position: static;
 }
 
+/**
+ * Class added to pause all aniamtions when the document is out of view.
+ */
+html.i-amphtml-animations-paused * {
```

Comment:
please add a comment stating that this does not work in iOS and reference 7846#issuecomment-293079217